### ISSUE TITLE
Peridot and Ring disagree on the best version of `ring-codec`

### ISSUE THREAD

**Issue by harold** [xeqi/peridot#49]
Hi! Thanks for peridot - we use it on every one of our ring projects for testing, and if it didn't exist, we'd have to start by making something much like it.

Recent updates to Peridot look great, but we noticed this transitive dep situation today:
```
$ lein deps :tree
Possibly confusing dependencies found:
[peridot "0.5.2"] -> [ring/ring-mock "0.3.2"] -> [ring/ring-codec "1.0.1"]
 overrides
[ring "1.7.0"] -> [ring/ring-jetty-adapter "1.7.0"] -> [ring/ring-servlet "1.7.0"] -> [ring/ring-core "1.7.0"] -> [ring/ring-codec "1.1.1"]
 and
[ring "1.7.0"] -> [ring/ring-servlet "1.7.0"] -> [ring/ring-core "1.7.0"] -> [ring/ring-codec "1.1.1"]
 and
[ring "1.7.0"] -> [ring/ring-jetty-adapter "1.7.0"] -> [ring/ring-core "1.7.0"] -> [ring/ring-codec "1.1.1"]
 and
[ring "1.7.0"] -> [ring/ring-devel "1.7.0"] -> [ring/ring-core "1.7.0"] -> [ring/ring-codec "1.1.1"]
 and
[ring "1.7.0"] -> [ring/ring-core "1.7.0"] -> [ring/ring-codec "1.1.1"]
````

It looks like the current (`0.4.0`) version of `ring-mock` does depend on the newer (`1.1.1`) version of `ring-codec`:
 - https://github.com/ring-clojure/ring-mock/blob/master/project.clj#L8

I believe that updating Peridot's `ring-mock` dep would align us all.

No worries. Hope this helps and that you have wonderful day.

**Comment by danielcompton** [comment]
Thanks, this is released in 0.5.3.

**Comment by harold** [comment]
Outstanding - thank you. 🙇
